### PR TITLE
Validate Tool

### PR DESCRIPTION
**Pull request by tschaffter**
See #39 

- This PR depends on #42

### CODE REVIEW

**Comment by tschaffter** [comment]
@thomasyu888 I've added a snippet of commented code that uses `jsonschema` package to validate a JSON object against a JSON schema. When testing a tool, we want to client to get the JSON object by sending a request to the Tool instead of reading JSON data from file. The snippet comes from this script that I used in our schemas repository to validate schemas: https://github.com/Sage-Bionetworks/nlp-sandbox-schemas/blob/develop/tools/validate/validate.py

**Comment by tschaffter** [comment]
'jsonschema>=3.2.0' is already in *setup.py*

**Comment by thomasyu888** [comment]
Thanks for starting this @tschaffter .  I was actually going to rename `NlpClient` to `DataNodeClient` and then have a `NlpToolClient`.  Thoughts?

File: nlpsandboxclient/validation.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+    def validate(self):
+        """Validate schema"""
+        # Check schema is correct first
+        if self.get_ui().status_code != 200:
+            raise ValueError("/ui endpoint not configured")
+        if self.get_health().get('status') != 'pass':
+            raise ValueError("/health endpoint not configured")
+        schema = self.get_tool()
+        Draft7Validator.check_schema(schema)
```
Comment (thomasyu888) [inline nlpsandboxclient/validation.py:26]: Is this schema the user specified schema?

**Comment by thomasyu888** [comment]
Depends on #42

File: nlpsandboxclient/validation.py (hunk 1 above)
Comment (tschaffter) [reply nlpsandboxclient/validation.py]: I think that you have figured this out in your next commit. The JSON Schema used to validate data will be available in https://github.com/Sage-Bionetworks/nlp-sandbox-schemas.

**Review by thomasyu888 — APPROVED** [review]

**Comment by thomasyu888** [comment]
@tschaffter, this can be merged

**Pull request merged**